config/config.php

```diff
@@ -154,7 +154,7 @@
 h1nzVu4H9N0bSFDgo65oIQ==',
 			],
 		],
-		'20.0.11' => [
+		'20.0.12' => [
```

Comment:
I think it might be time to remove this entirely and just let 

```php
		'20' => [
			'100' => [
				'latest' => '21.0.4',
				'internalVersion' => '21.0.4.1',
				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-21.0.4.zip',
				'web' => 'https://docs.nextcloud.com/server/21/admin_manual/maintenance/upgrade.html',
				'eol' => false,
				'minPHPVersion' => '7.3',
				'signature' => 'xm/qqcvl4auiDcnsDRGvegOMZNpkt6bo5m8oOa51WOeWYG1QKPvfJUTO7fgJvYcN
t7LXTG37rpCUVUR8apBqbizu4FjTkY8LKpQgV5xkxh9bQsaNEJCgnkiDUvxEYksi
/fLKYcf/xyKH3NXHMHU1t2NY+w5FHOZomh3ED0Mn4YLca4EVoZ7ABiUiV0o6N5GT
2xEEKaIq2tnmCnmp/1fcOwSTpNeJc5lTjFNIW5z7MS6y7iFjQIK6G+z+SYbq23sE
M+LjFGQ3UrLdLEUspKcF0awiacjCuvMGF6ns835tSzzZA9EhlsTIcfpO0rLWZo2B
h1nzVu4H9N0bSFDgo65oIQ==',
			],
		],
```